update: creating organizations with SAML (#864)
aiven/aiven-docs: docs/tools/aiven-console/howto/create-orgs-and-units.md

<Tabs groupId="group1">
<TabItem value="console" label="Console" default>

:::note

> You cannot create an organization if you logged in using
> an [identity provider](/docs/platform/howto/list-identity-providers). To create an
> organization log into the Aiven Console using another authentication method.

Reviewer: ```suggestion
You cannot create an organization while logged in with
an [identity provider](/docs/platform/howto/list-identity-providers). To create an
organization, log in to the Aiven Console using another authentication method.
```

</TabItem>
<TabItem value="terraform" label="Terraform">

:::note

> You cannot create an organization with a token that you created when you were logged in
> using an [identity provider](/docs/platform/howto/list-identity-providers).

Reviewer: ```suggestion
You cannot create an organization with a token created while logged in
with an [identity provider](/docs/platform/howto/list-identity-providers).
```
